pgjdbc-osgi-test/pom.xml

```diff
@@ -0,0 +1,205 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.postgresql</groupId>
+    <artifactId>pgjdbc-versions</artifactId>
+    <version>1.0.8</version>
+    <relativePath/>
+  </parent>
+
+  <artifactId>pgjdbc-osgi-test</artifactId>
+  <packaging>jar</packaging>
+  <name>PostgreSQL JDBC Driver - OSGi tests</name>
+  <version>42.3.0-SNAPSHOT</version>
+  <description>OSGi tests for java JDBC 4.2 (JRE 8+) driver for PostgreSQL database</description>
+  <url>https://github.com/pgjdbc/pgjdbc</url>
+
+  <licenses>
+    <license>
+      <name>The PostgreSQL License</name>
+      <url>https://www.postgresql.org/about/licence/</url>
+    </license>
+  </licenses>
+
+  <organization>
+    <name>PostgreSQL Global Development Group</name>
+    <url>https://jdbc.postgresql.org/</url>
+  </organization>
+
+  <properties>
+    <!-- Require JDK 1.8 or later -->
+    <current.jdk>1.8</current.jdk>
+    <javac.target>1.8</javac.target>
+    <pgjdbc.version>42.3.0</pgjdbc.version>
```

Comment:
should this match this project's version?

Reply:
it should be build.gradle.kts

Reply:
I used gradle to convert the maven pom. I may remove it as it doesn't really work anyway, or I can't figure out how to make it work

Reply:
I see. Then you probably forgot to add the generated build.gradle file to the commit.